ale/base/base.py

```diff
@@ -194,27 +235,51 @@ def sensor_model_version(self):
 
     @abc.abstractproperty
     def focal2pixel_lines(self):
+        """
+        Returns
+        -------
+        : int
+          Focal plane to detector lines
+        """
         pass
 
     @abc.abstractproperty
     def focal2pixel_samples(self):
+        """
+        Returns
+        -------
+        : int
+          Focal plane to detector samples
+        """
         pass
 
     @abc.abstractproperty
     def pixel2focal_x(self):
+        """
+        Returns
+        -------
+        : int
+          detector to focal plane x
+        """
         pass
 
     @abc.abstractproperty
     def pixel2focal_y(self):
+        """
+        Returns
+        -------
+        : int
+          detector to focal plane y
+        """
         pass
 
     @abc.abstractproperty
     def ephemeris_start_time(self):
         """
           Returns
         -------
-        : str
-          Start time of the image in UTC YYYY-MM-DDThh:mm:ss[.fff]
+        : double
+          Ephemeris start time of the image
```

Comment:
We should be more precise for these, they are "The start(stop) time of the image in ephemeris seconds past the J2000 epoch."